### ISSUE TITLE
Init node Adam/iter/Assign doesn't exist in graph

### ISSUE THREAD

**Issue by dizhanbin** [tensorflow/tensorflow#33907]
model = keras.Sequential([

        layers.Flatten(input_shape=(image_h, image_h, 1)),
    ])
model.compile(optimizer='adam',
                      loss='sparse_categorical_crossentropy',
                      metrics=['accuracy'])
model.save("model_right_error.h5")

--------------

converter = tf.lite.TFLiteConverter.from_keras_model_file("model_right_error.h5")
tflite_model = converter.convert()
open("model2.tflite", "wb").write(tflite_model)

tensorflow/core/grappler/grappler_item_builder.cc:637] Init node Adam/iter/Assign doesn't exist in graph

how to fix?

**Comment by dizhanbin** [comment]
android
does not encode a valid TensorFlowLite model: The model is not a valid Flatbuffer file

**Comment by oanush** [comment]
@dizhanbin ,
Hello, can you please refer this similar [Stackoverflow](https://stackoverflow.com/questions/56256478/converted-model-tflite-does-not-encode-a-valid-tensorflowlite-model-could-not-o) issue and let us know if it helped.Thanks!

**Comment by dizhanbin** [comment]
fixed it ,removed anaconda and installed package by pycharm.
